.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
-  - "0.8"
+  - "0.12"
```

Comment:
Can you switch this to `0.10`, `0.12`, `4.0`, `5.0` and `stable` as that's the current range we support